roles/openshift_health_checker/library/aos_version.py

```diff
@@ -47,8 +49,6 @@ def bail(error):  # pylint: disable=missing-docstring
 
     # determine what level of precision we're expecting for the version
     expected_version = module.params['version']
-    if expected_version.startswith('v'):  # v3.3 => 3.3
-        expected_version = expected_version[1:]
```

Comment:
The documentation for how to use a module should be in the module itself, not in a helper function from a different file that happens to be used in conjunction with the module through yet another file.